coreos-base/oem-hyperv/files/oem-release

```diff
@@ -1,4 +1,4 @@
 ID=hyperv
 VERSION_ID=@@OEM_VERSION_ID@@
 NAME="Microsoft Hyper-V"
-BUG_REPORT_URL="https://issues.flatcar-linux.org"
+BUG_REPORT_URL="https://issues.flatcar.org"
```

Comment:
Both work since we have a redirect in place when switched to the flatcar.org domain for the website, but it's good to use only one domain consistently